features-json/flexbox.json

```diff
@@ -73,7 +73,7 @@
       "description":"IE 11 does not vertically align items correctly when `min-height` is used [see bug](https://connect.microsoft.com/IE/feedback/details/816293/ie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center)"
     },
     {
-      "description":"IE 11 requires a unit to be added to the third argument, the flex-basis property [see MSFT documentation](https://msdn.microsoft.com/en-us/library/dn254946(v=vs.85).aspx)"
+      "description":"IE 11 requires a unit to be added to the third argument, the flex-basis property [see MSFT documentation](https://msdn.microsoft.com/en-us/library/dn254946\(v=vs.85\).aspx)"
```

Comment:
Use `%28` and `%29` instead of literal parens in the URL.
`\(` isn't valid JSON.